packages/compass-e2e-tests/helpers/commands/workspace-tabs.ts

```diff
@@ -0,0 +1,83 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+import type { WorkspaceTabSelectorOptions } from '../selectors';
+
+export async function navigateToMyQueries(browser: CompassBrowser) {
+  await browser.clickVisible(Selectors.SidebarMyQueriesTab);
+  await browser
+    .$(Selectors.workspaceTab({ title: 'My Queries', active: true }))
+    .waitForDisplayed();
+}
+
+async function closeTab(
+  browser: CompassBrowser,
+  selectorOptions: WorkspaceTabSelectorOptions,
+  autoConfirmTabClose: boolean
+): Promise<void> {
+  await browser
+    .$(Selectors.workspaceTab(selectorOptions))
+    .$(Selectors.CloseWorkspaceTab)
+    .click();
+
+  // wait until the tab goes away and if the confirmation modal opens, maybe confirm
+  await browser.waitUntil(async () => {
+    if (autoConfirmTabClose) {
+      // Tabs in "dirty" state can't be closed without confirmation
+      if (await browser.$(Selectors.ConfirmTabCloseModal).isExisting()) {
+        await browser.clickVisible(
+          browser.$(Selectors.ConfirmTabCloseModal).$('button=Close tab')
```

Comment:
`$('button=Close tab')`
TIL: Never saw this way of writing a selector before. Question - shouldn't Close tab be quoted?